integrationTests/relayers/ethToElrond_test.go

```diff
@@ -37,11 +38,11 @@ func TestRelayersShouldExecuteTransferFromEthToElrond(t *testing.T) {
 	ticker2 := "tck-000002"
 
 	value1 := big.NewInt(111111111)
-	destination1 := testsCommon.CreateRandomElrondAddress()
+	destination1 := testsCommon.CreateRandomElrondAddress().AddressBytes()
```

Comment:
why this change? If you would have kept the original form, the  L65 & L72 could have been:
```
destination1.AddressSlice()
```

Reply:
changed back